read-env-variable-system-property/solution.sh

```diff
@@ -6,3 +6,7 @@
 # 使得该java命令输出：
 # Environment variable AAA: $A;
 # System property BBB: '
+
+export AAA='$A'
+javac ReadEnvironmentVariableAndSystemProperty.java
+java -DBBB=' ReadEnvironmentVariableAndSystemProperty
```

Comment:
```suggestion
java -DBBB="'" ReadEnvironmentVariableAndSystemProperty
```

单独的一个单引号不是合法的命令，需要用双引号包裹起来